### PR TITLE
Explain why we do this

### PR DESCRIPTION
**Pull request by oliviertassinari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 
-This challenge is part of the hiring process at MUI for the React Engineer position.
+This challenge is part of the hiring process at MUI for some of the Software Engineer positions.
 The idea is to make as much progress as possible under a given time constraint (3-4 hours).
 
-## Context
+## Why are we doing this?
+
+At MUI, because we are a DevTools company, you'll make product decisions. You will flesh out product requirements and turn them into a technical design and implementation.
+This challenge simulates that.
```
Comment (oliviertassinari) [inline README.md:9]:
```suggestion
This challenge simulates that, it ranges from taking frontend product decisions down to diving into a complex codebase to implement a specific outcome.
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+This challenge simulates that. 
+We want to get a glimpse of how you will perform in the role.
+
+## Context about MUI
 
 In the short term, MUI is on a mission to become the UI toolkit for React. We are unifying the fragmented ecosystem of dependencies into a single set of simple, beautiful, consistent, and accessible React components.
 
-In the long term, MUI is on a mission to make building great React UIs and web applications quicker, simpler, and accessible to more people through low-code solutions. We resonate with this vision: https://youtu.be/GnO7D5UaDig?t=2451.
+In the long term, MUI is on a mission to make building great React UIs and web applications quicker, simpler, and accessible to more people through a low-code platform. It's about writing fewer code: https://youtu.be/GnO7D5UaDig?t=2451.
+
```
Comment (bharatkashyap) [inline README.md:17]: *It's about writing fewer lines of code or Its about writing lesser code?

**Comment by mbrookes** [comment]
https://www.youtube.com/watch?v=GyV_UG60dD4

File: README.md [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
+At MUI, because we are a DevTools company, you'll make product decisions. You will flesh out product requirements and turn them into a technical design and implementation.
+This challenge simulates that, it ranges from taking frontend product decisions down to diving into a complex codebase to implement a specific outcome.
+We want to get a glimpse of how you will perform in the role.
+
+## Context about MUI
 
 In the short term, MUI is on a mission to become the UI toolkit for React. We are unifying the fragmented ecosystem of dependencies into a single set of simple, beautiful, consistent, and accessible React components.
 
-In the long term, MUI is on a mission to make building great React UIs and web applications quicker, simpler, and accessible to more people through low-code solutions. We resonate with this vision: https://youtu.be/GnO7D5UaDig?t=2451.
+In the long term, MUI is on a mission to make building great React UIs and web applications quicker, simpler, and accessible to more people through a low-code platform. It's about writing fewer code: https://youtu.be/GnO7D5UaDig?t=2451.
```
Comment (oliviertassinari) [inline README.md:16]:
```suggestion
In the long term, MUI is on a mission to make building great React UIs and web applications quicker, simpler, and accessible to more people through a low-code platform. It's about writing less code: https://youtu.be/GnO7D5UaDig?t=2451.
```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,23 @@
...
 
-This challenge is part of the hiring process at MUI for the React Engineer position.
+This challenge is part of the hiring process at MUI for some of the Software Engineer positions.
 The idea is to make as much progress as possible under a given time constraint (3-4 hours).
 
-## Context
+## Why are we doing this?
+
+At MUI, because we are a DevTools company, you'll make product decisions. You will flesh out product requirements and turn them into a technical design and implementation.
+This challenge simulates that, it ranges from taking frontend product decisions down to diving into a complex codebase to implement a specific outcome.
```
Comment (oliviertassinari) [inline README.md:9]:
```suggestion
This challenge simulates that, we will review the product decisions you make, the quality of the code, as well as how you approach diving into a complex codebase.
```

**Comment by oliviertassinari** [comment]
> https://www.youtube.com/watch?v=GyV_UG60dD4

@mbrookes Do you mean that the terminology used is too broad, that it should be made more specific? Or explained in simpler terms?

File: README.md [hunk 5]
```diff
@@ -1,13 +1,23 @@
...
-This challenge is part of the hiring process at MUI for the React Engineer position.
+This challenge is part of the hiring process at MUI for some of the Software Engineer positions.
 The idea is to make as much progress as possible under a given time constraint (3-4 hours).
 
-## Context
+## Why are we doing this?
+
+At MUI, because we are a DevTools company, you'll make product decisions. You will flesh out product requirements and turn them into a technical design and implementation.
+This challenge simulates that, we will review the product decisions you make, the quality of the code, as well as how you approach diving into a complex codebase.
+We want to get a glimpse of how you will perform in the role.
```
Comment (oliviertassinari) [inline README.md:10]: Implicit

```suggestion
```

**Comment by mbrookes** [comment]
It was aimed at "MUI is on a mission" (I meant to comment on the specific line, and should perhaps have quoted the offending text for context). There's been an increase in use of corporate jargon internally and in public. It would be good to avoid it where possible.

**Comment by oliviertassinari** [comment]
Happy to hear an alternative. Would this work better?

```diff
-In the long term, MUI is on a mission to make building great React UIs and web applications quicker, simpler, and accessible to more people through a low-code platform. 
+The long-term mission of MUI is to make building great React UIs and web applications quicker, simpler, and accessible to more people through a low-code platform. 
```
